modules/media/jsx/columnFormatter.js

```diff
@@ -18,12 +18,19 @@ function formatColumn(column, cell, rowData, rowHeaders) {
     row[header] = rowData[index];
   }, this);
 
+  // create array of classes to be added to td tag
+  var classes = [];
+  if (row['Hide File'] === '1') {
+    classes.push("bg-danger");
+  }
+  // convert array to string, with blank space separator
+  classes = classes.join(" ");
```

Comment:
@ridz1208 suggested I do it this way, in case any other bootstrap elements need to be added to the table. Should I revert back to just an empty string or "bg-danger"?